components/Header.js

```diff
@@ -0,0 +1,22 @@
+import styled, { css } from "styled-components";
```

Comment:
you don't need to import { css } here, since you're not using it

Reply:
that was a leftover from a previous try and error phase :D thanks